README.md

```diff
@@ -25,6 +26,13 @@ You should use the template to create a new **private** repo in your org, for ex
 
 Once you've created a repository from the template, you'll want to `git clone` your new repo and `cd` into it locally.
 
+Edit the following values in Chart.yaml with your application information:
+- name
+- description
+- appVersion
+- home
+
+note: see Chart.yaml for examples
 
 
 ### 1. Install CLI
```

Comment:
Do we really need to keep the manual install instructions here in the Readme? Maybe it's a good thing to just link out to?